taxi/models.py

```diff
@@ -0,0 +1,41 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+from taxi_service import settings
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
```

Comment:
Consider adding a `verbose_name` attribute to your fields. This will provide a human-readable description of the field, which is useful for form labels and error messages.